app/konnect/runtime-manager/runtime-instances/gateway-runtime-aws.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Set up a Kong Gateway Runtime with AWS
+no_version: true
+content_type: how-to
+alpha: true
+---
+
+Set up a AWS runtime instance through the
+[{{site.konnect_short_name}} Runtime Manager](/konnect/runtime-manager) and
+configure your instance to accept configuration from
+{{site.konnect_short_name}}. The Runtime Manager keeps track of all runtime
+instances associated with the {{site.konnect_saas}} account.
+
+Runtime Manager provides a pre-populated template for a runtime instance in AWS. This template creates the following resources in AWS:
+* Amazon VPC along with internet gateway 
+* Secret
+* Amazon EC2 instances (key pair, role, profile)
+* Auto Scaling group
+* Network Load Balancer
+* Optional: CloudWatch log group 
+* Optional: Redis 
+
+{:.note}
+> **Note:** Kong does not host runtimes. You must install and host your own
+runtime instances.
+
+## Prerequisites
+
+* An account with AWS that is already configured
+* An account that can use IAM resources
+
+## Configure the AWS Gateway instance
+
+1. Open the {% konnect_icon runtimes %} **Runtime Manager**.
+
+1. Select a runtime group.
+
+1. Click **New Runtime Instance**.
+
+1. Click **AWS**.
+
+1. From **AWS Region**, select the region where you want to host the runtime instance.
+
+1. Configure the fields on the page:
+
+    | Field | Description |
+    |---|---|
+    | AWS Region | The AWS region where the CloudFormation template should be created. |
+    | Instance type for the gateway runtime instances | The instance type to use for the runtime instance. This must be a valid x86_64 EC2 instance type. You can edit this later in AWS. |
```

Comment:
> The instance type to use for the runtime instance.

I know this is from the tooltip, but this sentence tells me nothing. Can we simplify to just:

```suggestion
    | Instance type for the gateway runtime instances | A valid x86_64 EC2 instance type. You can edit this later in AWS. |
```